src/qt/publishdialog.cpp

```diff
@@ -158,6 +158,21 @@ void PublishDialog::on_button_OkCancel_accepted()
         case 1: SP->m_dw_qos.durability().kind = eprosima::fastdds::dds::TRANSIENT_LOCAL_DURABILITY_QOS; break;
     }
 
+    std::cout<<"Create Shape publisher"<<std::endl;
```

Comment:
After we checked that the functionality is okey, please remove these debug lines